test/app.js

```diff
@@ -177,27 +187,26 @@ module.exports = _.defaultsDeep({
         require('../') // trailpack-sequelize
       ]
     },
-    database: {
-      stores: {
-        teststore: {
-          host: 'localhost',
-          dialect: 'sqlite',
-          storage: './test/test.sqlite',
-          database: 'test'
-        },
-        storeoverride: {
-          host: 'localhost',
-          dialect: 'sqlite',
-          storage: './test/test.sqlite',
-          database: 'test'
-        },
-        uristore: {
-          uri: 'sqlite://testuser:password@testhost:1234/testdb'
-        }
-      },
-      models: {
-        defaultStore: 'teststore',
+    models: {
```

Comment:
so now models configuration are directly under a models.js file ? for 2 fields ? :(

Reply:
So here's my rationale for this.  `config/database.js` is gone in v3.  The "stores" that was inside of it was moved out to `config/stores`, the problem being that the defaults are now gone for models.  So I just moved models outside of it as well.  I think this may have been in oversight on trails v3.  The example I was using trailpack-knex does not even support defaults for the stores, but we have to have defaults for quite a few trailpacks that depend on sequelize.  This was kind of a compromise.

Reply:
I've updated this now so that we can configure model overrides from inside the `config/models`, which makes it much more useful I think, and a feature I would love to use in many of my trailpacks.